### DISCUSSION TITLE
Magick.exe vs convert.exe syntax issue?

### DISCUSSION THREAD

**Discussion by zBernie** [ImageMagick/ImageMagick#4964] [Help]
The command below worked previously using the "convert" command.  But I get the error shown when I try using the magick.exe command. How can I execute this command using magick.exe?   I just upgraded to IM 7.1.0.

magick $jpgfile -font Arial -pointsize 60 -fill lightblue -quality 100 -gravity southeast ` -auto-orient -annotate +80+50 %[exif:DateTimeOriginal] $jpgfile

magick.exe: unknown image property "%[exif:DateTimeOriginal]" @ warning/property.c/InterpretImageProperties/4209.

**Comment by fmw42** [comment]
Please show what you get from 

`magick.exe -version`

and also identify your platform/OS

**Reply by fmw42** [reply]
Also post an image the does not work.

**Reply by zBernie** [reply]
This is Windows 10.  Here's the version info:

PS C:\Program Files\ImageMagick-7.1.0-Q16-HDRI> magick -version
Version: ImageMagick 7.1.0-27 Q16-HDRI x64 2022-03-05 https://imagemagick.org
Copyright: (C) 1999 ImageMagick Studio LLC
License: https://imagemagick.org/script/license.php
Features: Cipher DPC HDRI Modules OpenCL OpenMP(2.0)
Delegates (built-in): bzlib cairo flif freetype gslib heic jng jp2 jpeg jxl lcms lqr lzma openexr pangocairo png ps raqm raw rsvg tiff webp xml zip zlib
Compiler: Visual Studio 2022 (193131104)

**Reply by fmw42** [reply]
Run your command without Powershell?  Does it work, when you specify your images directly, not as variables?

**Reply by fmw42** [reply]
What version of Powershell?

**Reply by zBernie** [reply]
Yes it works outside of powershell, I should have tried that but just assumed it was a syntax discrepancy between convert.exe and magick.exe.   I'm using PowerShell 7.2.0-preview.4, the same version I've been using for many months.   Maybe I'll update my version of powershell.   Thanks for your help.

**Comment by snibgo** [comment]
```
-gravity southeast ` -auto-orient
```
Why is there a back-tick?

```
unknown image property "%[exif:DateTimeOriginal]" 
```
You will get that warning if your image does not contain that metadata.

**Reply by zBernie** [reply]
That's a line continuation character in PowerShell.  Actually, the command in the script is now working.  Not sure why it was not working yesterday.

**Reply by fmw42** [reply]
Please always format your code with the {} button after highlighting it so that line continuations are properly shown in this forum and not confuse others.
